package.json

```diff
@@ -22,16 +24,15 @@
   "devDependencies": {
     "@swc/cli": "^0.1.62",
     "@swc/core": "^1.3.76",
+    "@types/express": "^4.17.17",
     "@types/jest": "^29.5.3",
     "@types/node": "^20.4.10",
     "@types/node-schedule": "^2.1.0",
     "jest": "^29.6.2",
     "ts-jest": "^29.0.0-next.1",
     "ts-node": "^10.9.1",
+    "ts-node-dev": "^2.0.0",
     "tsconfig-paths": "^4.2.0",
     "typescript": "^5.1.6"
-  },
-  "_moduleAliases": {
-    "@milanbot": ".build/src"
```

Comment:
Absolutely! Forgot to put it back when I was messing around with the build (see also below). [cyclic.sh](https://cyclic.sh) might need `index.js` at the root (and not in a subfolder), and I was trying to avoid leaking package.json too. I've now created `version.ts` which gets updated pre build with the `version` from `package.json`.